src/components/MoreInfoModal/MoreInfoModal.tsx

```diff
@@ -72,9 +72,10 @@ export default function BasicModal() {
       <Modal
         open={open}
         onClose={handleClose}
-        aria-labelledby="modal-modal-title"
-        aria-describedby="modal-modal-description"
+        aria-labelledby="more-info-modal"
+        aria-describedby="this-modal-contains-house-info"
```

Comment:
Both the `aria` properties should point to ids of elements that label/describe a given element.

Reply:
Ok
I may have a brain fart here but what should the text be then? 😂

Reply:
There should be the id(s) of other element(s) (separated by spaces). Take a look at this: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby. MUI might have its own way of handling this, not sure about that. You could also consider skipping it.

Reply:
Ok, I skipped it ;)